gm4_villager_telltale/data/gm4/advancements/vt_find_workstation.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
This advancement should be named more consistently with the rest of GM4 modules. Modules with only a single advancement should be named their own module name. (So this should be renamed to `villager_telltale`)